examples/resources/saml-connector.yaml

```
@@ -12,8 +12,11 @@ spec:
   # button on the cluster login screen so it will say "Login with Okta"
   display: Okta
   # SAML provider will make a callback to this URL after successful authentication
-  # cluster-url is the address the cluster UI is reachable at
-  acs: https://<cluster-url>/v1/webapi/saml/acs
+  # cluster-url is the address the cluster UI is reachable at.
```

```suggestion
  # acs is the address the cluster UI is reachable at.
```

Do we mean to say `acs` here? I don't see anything called `cluster-url`.

---

The first part of the URL still says `<cluster-url>` as it needs to be filled in.
Here's how it looks in my editor:
```yaml
# SAML provider will make a callback to this URL after successful authentication
  # cluster-url is the address the cluster UI is reachable at.
  # The last segment of the URL must be identical to the connector metadata name.
  acs: https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector```

---

Ahh, totally missed that. Thanks!